google-cloud-trace/lib/google/cloud/trace/v1/doc/google/protobuf/timestamp.rb

```diff
@@ -0,0 +1,83 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Google
+  module Protobuf
+    # A Timestamp represents a point in time independent of any time zone
```

Comment:
Sorry, let me clarify a bit. These dummy class for documentation purpose will be used as the source of the documentation for GRPC messages in both gem (rubydoc.info) and gh-pages site.